src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -72,8 +72,21 @@ Run ``npm run test`` to execute the unit tests via `Karma
 Running End-to-End Tests
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Run ``npm run e2e`` to execute the end-to-end tests via
-`Protractor <http://www.protractortest.org/>`__.
+Run ``npm run e2e -- --serve=false --base-href <DASHBOARD_URL>`` to execute the
+end-to-end tests via `Protractor <http://www.protractortest.org/>`__.
+
+Note:
+  You need Chrome installed and to have the dashboard frontend built.
+
+Docker:
+  Alternatively, if you don't have/want Chrome installed in your system,
+  you can use a docker image to run the e2e tests.
+
+  .. code:: bash
+
+    # docker run -d -v $(pwd):/workdir --net=host --name angular-e2e-container rogargon/angular-e2e
```

Comment:
Do these commands have to be run as the root user? If not, consider replacing the command prompt with `$` instead.

Reply:
docker commands require root.